templates/hotgridPopupToolbar.jsx

```diff
@@ -67,7 +67,7 @@ export default function HotgridPopupToolbar(props) {
       }
 
       <button
-        className="btn-icon hotgrid-popup__close"
+        className="btn-icon notify__btn-icon hotgrid-popup__close"
```

Comment:
See here:

https://github.com/adaptlearning/adapt-contrib-vanilla/pull/448#discussion_r1235090469

Reply:
Currently we have `notify__btn-icon next` & `notify__btn-icon back` So, for consistency and to assist with [this](https://github.com/adaptlearning/adapt-contrib-vanilla/pull/448#discussion_r1235090469) perhaps add `close` here? 

That way we can still use the same class/selector and can target the different button types separately. 

Note, I think these class naming is off and should perhaps be is-next / is-previous & is-close. Another reason to rationalise these button classes further!

Reply:
For the purpose of separating the styles for [controls and close buttons](https://github.com/adaptlearning/adapt-contrib-vanilla/pull/448#discussion_r1235090469) we could just do this targeting the existing selectors? See below

```
&__controls {
  .notify-btn-icon;
}
&__close {
  .notify-btn-icon;
}
```
I do agree the class naming is off here and I'm in favour of a shared selector e.g. `.notify__btn-icon` and the tag on button type selectors e.g. `is-next` etc but this will need addressing across adapt for consistency so perhaps it's best to address this as part of the [Adapt button issue](https://github.com/adaptlearning/adapt-contrib-core/issues/386). As in capture all button types, agree terminology so we don't have to keep changing templates and then roll out. Might be easier to keep track of consistency that way too.

Reply:
I was thinking that stay on target with `notify__btn-icon`. so not update `next`/`back` and just use:

```
&__btn-icon next, &__btn-icon back {
  .notify-btn-icon;
}

&__btn-icon close {
  .notify-btn-icon;
}
```

then when it comes to addressing the buttons in that [issue](https://github.com/adaptlearning/adapt-contrib-core/issues/386) I think perhaps this is a longer term problem that has many complexities - but perhaps we'd have adapt utility classes like 'primary-button' & 'secondary-button' (or whatever so they'd be called) in addition to the BEM naming already there. So the `is-` notation is sort of a separate issue, was just pointing out the problems! 

I maybe getting confused here but aren't there only a small handful of places where this would need to change?

Reply:
> I maybe getting confused here but aren't there only a small handful of places where this would need to change?

This applies to the following plugins/templates: 
Controls: Hotgrid and Hotgraphic
Close: Notify popup, Notify push, Hotgraphic, Hotgrid and Tutor.

The existing PR satisfies the [issue](https://github.com/cgkineo/adapt-hotgrid/issues/126) raised, to reduce duplicate code. 

Based on suggestions above, I think next steps is to confirm the class naming for [Adapt buttons](https://github.com/adaptlearning/adapt-contrib-core/issues/386) and then roll out to plugins rather than tackling in this PR.